manifest.json

```diff
@@ -1,10 +1,10 @@
 {
-	"id": "obsidian-functionplot",
-	"name": "Obsidian Functionplot",
-	"minAppVersion": "0.11.13",
-	"description": "A plugin for displaying mathematical graphs in obsidian.md.",
-	"author": "leonhma",
-	"authorUrl": "https://github.com/leonhma",
-	"isDesktopOnly": false,
-	"version": "1.1.0"
-}
\ No newline at end of file
+  "id": "obsidian-functionplot",
+  "name": "Obsidian Functionplot",
+  "minAppVersion": "0.11.13",
+  "description": "A plugin for displaying mathematical graphs in obsidian.md.",
+  "author": "leonhma",
+  "authorUrl": "https://github.com/leonhma",
+  "isDesktopOnly": false,
+  "version": "1.1.1"
```

Comment:
you dont even have to bump the version yourself, the release workflow will take care of that

Reply:
Good to know, it's been taken care of.